src/ExtensionsSample/Samples/ApiHubSamples.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.Azure.WebJobs.Extensions.ApiHub;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExtensionsSample.Samples
+{
+    public static class ApiHubSamples
```

Comment:
All the code in this file needs fixed up formatting wise. It also doesn't look complete